xpk.py

```diff
@@ -3091,7 +3511,7 @@ def directory_path_type(value):
         'but if your docker container is missing the dependencies, it might '
         'look more like "--command=\'bash setup.sh && python3 train.py\'".'
     ),
-    required=True,
+    required=False,
```

Comment:
When is --command not needed?

Small edge case:
We might need a if statement check in xpk to check that command is set if --use-pathways is not provided.